test/csharp/csharp.nuget.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
@Sergei Vorobev good catch. I removed the file. Thanks!

---
In reply to: [78430315](https://github.com/Powershell/Powershell/pull/2160#discussion_r78430315) [](ancestors = 78430315)